skhep/math/kinematics.py

```diff
@@ -47,6 +50,64 @@ def Kallen_function(x, y, z):
     """
     return (x - y - z) ** 2 - 4 * y * z  # faster to evaluate condensed form used
 
+def Armenteros_Podolanski_variables(pplus_3Dvec, pminus_3Dvec):
```

Comment:
[There should be two black lines between top level functions](https://www.python.org/dev/peps/pep-0008/#blank-lines) like the other functions in this file.